### PR TITLE
GOG automatic import

### PR DESCRIPTION
**Pull request by imLinguin**
That's a lot of data  
we still need to go through that data to double check the new entries are correct

### CODE REVIEW

File: ULWGL-database.csv [hunk 1]
```diff
@@ -587,3 +587,4730 @@ Soldier of Fortune II: Double Helix - Gold Edition,gog,1228964594,ulwgl-12289645
...
+Ziggurat 2,gog,1427459497,ulwgl-1159560,,
+The Book of Bondmaids,gog,1635274144,ulwgl-1398070,,
+Bite the Bullet,gog,1213517212,ulwgl-1075550,,
+Defender's Quest 2: Mists of Ruin,gog,1988808085,ulwgl-252190,,
+Hard West,gog,1442827661,ulwgl-307670,,
+The Textorcist: The Story of Ray Bibbia Demo,gog,1093291391,ulwgl-940680,,
+Cloud Gardens,gog,1107152216,ulwgl-1372320,,
+Night in the Woods: Weird Autumn Edition,gog,1330400591,ulwgl-481510,,
+B.I.O.T.A.,gog,1321782603,ulwgl-1640320,,
+The Legend of Heroes: Trails in the Sky SC,gog,1444826419,ulwgl-251290,,
```
Comment (keenanweaver) [inline ULWGL-database.csv:2839]: Looks like my ID from https://github.com/Open-Wine-Components/ULWGL-database/pull/18 is wrong. No idea where I got that number. So you can remove line 583.

File: ULWGL-database.csv [hunk 2]
```diff
@@ -587,3 +587,4730 @@ Soldier of Fortune II: Double Helix - Gold Edition,gog,1228964594,ulwgl-12289645
...
+Singularity™,gog,1207659239,ulwgl-42670,,
+FORCED SHOWDOWN,gog,1453126104,ulwgl-265000,,
+Below the Stone,gog,1235841495,ulwgl-1170230,,
+Steel Division 2,gog,1911255510,ulwgl-919640,,
+Jade Empire: Special Edition,gog,1207659237,ulwgl-7110,,
+SKYHILL,gog,1458729425,ulwgl-382140,,
+The Adventure Pals,gog,1391555008,ulwgl-396710,,
+Leap of Love,gog,1111421371,ulwgl-1469300,,
+Inherit the Earth,gog,1207659231,ulwgl-257260,,
+Alone in the Dark 3,gog,1207660973,ulwgl-548900,,
```
Comment (keenanweaver) [inline ULWGL-database.csv:1221]: How will we be handling DOSBox games? Are they in scope?

**Review by keenanweaver — COMMENTED** [review]
This is missing: https://www.gogdb.org/product/1828104558

Also, I'm a pretty big GOG whale, so I'm willing to test any fixes.

Screenshot_20240224_204147

File: ULWGL-database.csv (hunk 2 above)
Comment (imLinguin) [reply ULWGL-database.csv:1221]: I guess I can add filtering for those, each launcher will probably try to use native dosbox version

Comment (keenanweaver) [reply ULWGL-database.csv:1221]: I'd assume the same for ScummVM games? I don't think GOG uses any other tools besides DOSBox & ScummVM for releases.

**Comment by imLinguin** [comment]
I think it's better if we add entries as we need them to avoid any potential issues coming from auto imports

**Pull request closed without merging**